designs/developer/topology-45/index.md

```diff
@@ -0,0 +1,300 @@
+
+![Select Application Grouping](../topology-43/img/Select_app_group.png)
+
+A modal opens up allowing the user to select an application from a dropdown menu.
+
+![Create Application Grouping](../topology-43/img/Edit_app_grouping_new.png)
+
+Users get an option to select from a list of pre-existing application, or create a new one to add the component to.
+To create a new application, select the option ‘Create Application Group’
+
+![Create Application Grouping](../topology-43/img/Edit_application_grouping_dialogue.png)
+
+A new field appear to allow users to enter a name for the new application group to be created.  
+
+![Connector Created](../topology-43/img/connector_created.png)
+
+On clicking on ‘Save’’ the component gets added to the newly created application group.  
+
+##### User wants to delete a component
+
+Delete using action menu on the side panel
+
+![Side Panel Menu](../topology-43/img/application_editgrouping_menu.png)
+
+On selecting a node, the side panel appears with information about the node. The actions dropdown on the side panel include node specific actions.  
+
+![Delete Component](../topology-43/img/delete_component.png)
+
+Select Delete Component
+Or, hit ‘Delete’ key after selection.  
+
+![Delete Confirmation](../topology-43/img/node_delete_confirmation.png)
+
+A delete confirmation is displayed.  
+
+![Node Deleted](../topology-43/img/image69.png)
+
+Upon confirming, the node and any associated connectors disappear  
+
+##### User wants to add a component node to an application group using drag and drop
+
+![Click to drag](../topology-43/img/node_select.png)
+
+Click and drag the node to the desired application.  
+
+![Drag to move](../topology-43/img/node_drag_newapp.png)
+
+While dragging, the other components become faded in appearance and the App group in contact with the moving node gets a hover state.
+
+![Drag to move](../topology-43/img/moved_node.png)
+
+On release the component gets added to the application group.
+
+##### User wants to move a component node from one app to another
+
+![Drag to change application](../topology-43/img/image68.png)
+
+Click and drag the node to the desired application.
+
+![Drop over new application](../topology-43/img/application_change_grouping.png)
+
+Drop the component node inside another app by releasing the click.
+
+![Confirm Action](../topology-43/img/node_move_dialogue.png)
+
+On dropping the node, a dialogue box appears to confirm the action of moving Node1 from App1 to App2.
+
+![Node Moved](../topology-43/img/image39.png)
+
+Once confirmed, the component node becomes a part of the new application group and the group elements rearrange themselves to accommodate the change.Exception: Moving from inside an application group to no group.  
+
+### 2. Actions For Application Groups
+
+#### [Collapse Application Groups](openshift-origin-design/designs/developer/topology-44/topology-grouping.md)
+#### Delete Application Group
```

Comment:
Looks like the UI now says "Delete Application", which is consistent with the Application selector.  You may want to make these changes here ( or in a follow up PR )

Reply:
Changed!